src/calendar/CalendarMonth.jsx

```diff
@@ -32,6 +26,12 @@ const CalendarMonth = React.createClass({
     value: CustomPropTypes.momentOrMomentRange,
   },
 
+  componentWillMount() {
+    const lang = moment().localeData();
```

Comment:
I think making this prop-driven falls more in line with React's functional paradigm. I feel like it's better for the component to rely on either state passed into it (`props`) or its own internal state (`state`) and not rely on global state, like `moment().localeData()`. Also, if the moment locale changes, I don't think it would trigger an update of this component.

Thoughts?